asciidoc/courses/app-java/course.adoc

```diff
@@ -11,35 +11,40 @@
 :package: neoflix
 // end::config[]
 
-== Course Description
+== Course Objectives
 
-In this course, we walk through the steps to integrate Neo4j into your Java project.
+In this course, you will learn how to use Neo4j in your Java project. You will understand how to map a graph data model to a Java application domain model.
 
-We will follow the steps to create an instance of the Neo4j Java Driver, learn about sessions and transactions, and use that knowledge to build a REST API with link:https://javalin.io/[Javalin^] that will work with a pre-built Single Page Application (SPA).
-
-The course automatically creates a new `{usecase}` sandbox within link:https://sandbox.neo4j.com/?usecase={usecase}[Neo4j Sandbox] that you will use throughout the course.
+You will learn the skills you need to build applications with Java to read data from and write data to Neo4j.
 
 === Prerequisites
 
-By taking this course, we assume that you have a working knowledge of Java, Maven and the Java ecosystem.
-We also assume that you have at least a basic knowledge of Neo4j.
+Taking this course, you should have the following previous knowledge/skills:
+
+1. Be able to write Java programs.
+2. Have completed the link:/courses/neo4j-fundamentals/[Neo4j Fundamentals] and link:/courses/cypher-fundamentals/[Cypher Fundamentals] courses.
 
-If you haven't already done so, we recommend that you also take the link:/courses/neo4j-fundamentals[Neo4j Fundamentals] course in order to gain a basic understanding of Neo4j and the link:/courses/cypher-fundamentals[Cypher Fundamentals] to understand how to query Neo4j using Cypher.
+include::{shared}/courses/gitpod/overview.adoc[leveloffset=+1]
 
 === Duration
 
-8 hours
+4 hours
 
 === What you will learn
 
-* The lifecycle of the Neo4j Driver and how it relates to your application
-* How to install and instantiate the Neo4j Java Driver into your Java project
-* How read and write transactions work with Neo4j
-* Best practices on how to use Neo4j within your Java project
+* How to use the Neo4j with Java
+* How to map a graph data model to a Java application domain model
+* How to convert between Neo4j and Java data type systems
+* How to read data from and write data to Neo4j
+* Continuing education: ideas for further development and where to find resources
+* Optional:
+** Authentication with link:https://github.com/auth0/auth0-java[Auth0^] and link:https://jwt.io/[JWT Tokens^]
+** Passwords encrypted and verified with link:https://javadoc.io/doc/at.favre.lib/bcrypt/latest/index.html[bcrypt^]
+** Testing using link:https://junit.org/junit5/[JUnit5^]
 
 [.includes]
 == This course includes
-
+//TODO: Update this!!
```

Comment:
I think this might be my fault. There are some changes I started to make that look like they're in here....

Reply:
Sorry I didnt notice them.